### PR TITLE
Bump version 0.1.2

### PR DESCRIPTION
**Pull request by eguchi1904**
## Check list

- [x] I wrote [CHANGELOG.md](./CHANGELOG.md) if the pull request adds/modifies features.

## Summary

Loosen the version restriction of `PyGObject` and release 0.1.2.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.1.2 (2021-09-06)
+
+- Loosen the version restriction of `PyGObject` to be compatible with a `python3-gi` package provided by apt in Jetson application's base image.
```
Comment (kenoss) [inline CHANGELOG.md:5]:
```suggestion
- Loosen the version requirement of `PyGObject` to be compatible with a `python3-gi` package provided by apt in Jetson application's base image.
```

I think it is better to use the words requirement or constraint.

**Review by kenoss — APPROVED** [review]
LGTM with nitpicks

**Comment by eguchi1904** [comment]
@kenoss The approve was automatically dismissed by new commit. Could you re-approve this PR?

**Review by kenoss — APPROVED** [review]
Thanks! :100:

**Comment by eguchi1904** [comment]
:pray:

**Pull request merged**